config/core/test-configs.yaml

```diff
@@ -2842,6 +2842,11 @@ test_configs:
   - device_type: sun50i-h5-libretech-all-h3-cc
     test_plans:
       - baseline
+      - baseline-nfs
+      - kselftest-alsa
+      - kselftest-kvm
+      - kselftest-rtc
+      - ltp-crypto
```

Comment:
Passing in staging: https://staging.kernelci.org/test/plan/id/62a9a58786099cf0ef74f1f9/